services/m3aggregator/config/aggregator.go

```diff
@@ -303,7 +317,7 @@ func (c *AggregatorConfiguration) NewAggregatorOptions(
 	entryPoolOpts := c.EntryPool.NewObjectPoolOptions(iOpts)
 	entryPool := aggregator.NewEntryPool(entryPoolOpts)
 	opts = opts.SetEntryPool(entryPool)
-	entryPool.Init(func() *aggregator.Entry { return aggregator.NewEntry(nil, opts) })
+	entryPool.Init(func() *aggregator.Entry { return aggregator.NewEntry(nil, initRuntimeOpts, opts) })
```

Comment:
pass in runtimeOptsManager.RuntimeOptions() here instead of initRuntimeOpts?

Reply:
Sure sounds good.